test/core/util/fuzzer_corpus_test.cc

```diff
@@ -48,15 +48,21 @@ DEFINE_string(directory, "", "Use this directory as test data");
 class FuzzerCorpusTest : public ::testing::TestWithParam<std::string> {};
 
 TEST_P(FuzzerCorpusTest, RunOneExample) {
+  grpc_init();
   gpr_log(GPR_DEBUG, "Example file: %s", GetParam().c_str());
   grpc_slice buffer;
   squelch = false;
   leak_check = false;
   GPR_ASSERT(GRPC_LOG_IF_ERROR("load_file",
                                grpc_load_file(GetParam().c_str(), 0, &buffer)));
-  LLVMFuzzerTestOneInput(GRPC_SLICE_START_PTR(buffer),
-                         GRPC_SLICE_LENGTH(buffer));
+  uint8_t* data;
```

Comment:
Why do we need to copy the data out of the slice here?  Why not simply wait until after `LLVMFuzzerTestOneInput()` returns before calling `grpc_shutdown_blocking()`?

Reply:
Because the fuzzer test will do its own `grpc_init` and `grpc_shutdown_blocking`, which is part of the test. gRPC needs to be fully shutdown when a test start.